doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -51,7 +51,44 @@ TODO
 
 ### Riak
 
-TODO
+Changes to Riak schema are backwards compatible with the current MongooseIM release.
+This means that in case of skipping of the migrations, only some new features (namely GDPR data retrival) won't work correctly.
+
+#### Step 1
+
+Please update the Riak schema:
+```bash
+# Set the RIAK_HOST to your Riak HTTP endpoint
+# Set the RIAK_MAM_SCHEMA_PATH to point to new schema path, which
+# by default is: RIAK_MAM_SCHEMA_PATH=tools/mam_search_schema.xml
+curl -v -XPUT $RIAK_HOST/search/schema/mam \
+    -H 'Content-Type:application/xml' \
+    --data-binary @${RIAK_MAM_SCHEMA_PATH}
+```
+
+After that we need to either reload all Riak nodes (restart them) or manually reload the schema on live nodes.
+Reloading of schema of live nodes requires access to Erlang Shell of one of the Riak nodes (any of them).
```

Comment:
Reloading the schema on live nodes requires access to Erlang Shell of one of the Riak nodes (any of them).